core/src/main/java/org/eclipse/hono/util/RegistrationResult.java

```diff
@@ -18,14 +18,9 @@
  * A container for the result returned by Hono's registration API.
  *
  */
-public final class RegistrationResult {
-
-    private final int status;
-    private final JsonObject payload;
-
+public final class RegistrationResult extends RequestResponseResult {
```

Comment:
Formally not (could be substituted by RequestResponseResult), but I thought it is better to keep a specific result class per API available to have an extension point available for each API.

I propose to keep it like that, but there are no pure technical reasons for it.
WDYT?